### PR TITLE
revamped about page

### LINKED ISSUE #2004: Enhance about page
Enhance about page with following details 

Mentors
Core team members 
Alumni
Contributers (as it is)

For inspiration

https://www.rust-lang.org/governance/teams/core
https://rubyonrails.org/community/

### PR DESCRIPTION
**Pull request by Kartikay304**
Fixed #2004 

#### Describe the changes you have made in this PR -
1. Modified about_controller index method for member name & images placeholder.
2. Made hardcore about page for related about the view.
3. Added styles and hover effects.
4. Added manual slideshow to switch between Mentors, Core team members & Alumni

### Screenshots of the changes (If any) -
Before -
04 09 2021_02 04 12_REC

After -
04 09 2021_15 21 46_REC

@tachyons @nitin10s, will you please look at my PR.

### CODE REVIEW

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -70,3 +156,28 @@
   }
   getContributors('https://api.github.com/repos/CircuitVerse/CircuitVerse/contributors');
 </script>
+<script>
```
Comment (tachyons) [inline app/views/about/index.html.erb:159]: No need of slides, just have a static page

**Comment by tachyons** [comment]
Also include full page in screenshot

File: app/views/about/index.html.erb (hunk 1 above)
Comment (Kartikay304) [reply app/views/about/index.html.erb:159]: And what about the overall design of the core team, mentor, alumni card? Is it good or, should the design change?

Comment (tachyons) [reply app/views/about/index.html.erb:159]: Please include full screenshot , cc @nitin10s

Comment (Kartikay304) [reply app/views/about/index.html.erb:159]: 04 09 2021_15 21 46_REC

**Review by nitin10s — CHANGES_REQUESTED** [review]
Yes, the slides are confusing, it seems like there are more members in single categories rather than different categories in each slide.

Also, the design looks very big, what we can do is have them in circles with text(name) below the image. We can aim for 4/5 in one row. This way, they'll be uniform with the contributor's design (which is also in circles). Whatsay?

**Comment by Kartikay304** [comment]
> Yes, the slides are confusing, it seems like there are more members in single categories rather than different categories in each slide.
> 
> Also, the design looks very big, what we can do is have them in circles with text(name) below the image. We can aim for 4/5 in one row. This way, they'll be uniform with the contributor's design (which is also in circles). Whatsay?

Yeah! We can make them in circles with text below the image. But there is  one feature request to make HUMANS OF CIRCUITVERSE PAGE(issue #2005 ). If can make one, then there will be no need for contributor section in ABOUT PAGE.I already made it and  temporarily deployed the  page at  [THIS](http://voracious-distribution.surge.sh/index.html) url. Can you take a look at it and if you would like some other changes and enhancements, please let me know.

Here my suggestion request! on how to give contributors to enter their details manually.
1>.To give contributors options to enter their details manually. I thought about creating an admin/contributor page which has.
Two types of users i) admin and ii) contributors.
2>.Those with admin login access can add/manage users (contributors) and can also approve/edit the post (details which
Contributors want to add to their page).
3>. And contributors have access to add a post or edit their previous post.

I have  completed the front-end of the admin/contributor login page.
For now, which can be accessed from the [DASHBOARD](http://voracious-distribution.surge.sh/admin/sidenav.html) option on Navigation bar.

**Comment by Kartikay304** [comment]
> > Yes, the slides are confusing, it seems like there are more members in single categories rather than different categories in each slide.
> > Also, the design looks very big, what we can do is have them in circles with text(name) below the image. We can aim for 4/5 in one row. This way, they'll be uniform with the contributor's design (which is also in circles). Whatsay?
> 
> Yeah! We can make them in circles with text below the image. But there is one feature request to make HUMANS OF CIRCUITVERSE PAGE(issue #2005 ). If can make one, then there will be no need for contributor section in ABOUT PAGE.I already made it and temporarily deployed the page at [THIS](http://voracious-distribution.surge.sh/index.html) url. Can you take a look at it and if you would like some other changes and enhancements, please let me know.
> 
> Here my suggestion request! on how to give contributors to enter their details manually.
> 1>.To give contributors options to enter their details manually. I thought about creating an admin/contributor page which has.
> Two types of users i) admin and ii) contributors.
> 2>.Those with admin login access can add/manage users (contributors) and can also approve/edit the post (details which
> Contributors want to add to their page).
> 3>. And contributors have access to add a post or edit their previous post.
> 
> I have completed the front-end of the admin/contributor login page.
> For now, which can be accessed from the [DASHBOARD](http://voracious-distribution.surge.sh/admin/sidenav.html) option on Navigation bar.

@nitin10s

**Pull request closed without merging**